db/migrate/20190525141132_category_id_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class CategoryIdToArticles < ActiveRecord::Migration[5.2]
```

Comment:
the same thing is reffered on the first migration file also. any reason for that?